examples/rules.dl

```diff
@@ -0,0 +1,48 @@
+// Is this symbol an access path to some data?
+.decl isAccessPath(x: symbol)
+// Is this symbol a label.
+.decl isLabel(x: symbol)
+// A data flow edge
+.decl edge(src: symbol, tgt: symbol)
+ // A data flow path
+.decl path(src: symbol, tgt: symbol)
+// Definitely has this label.
+.decl hasLabel(v: symbol, label: symbol)
+// Definitely has this label.
+.decl claimHasLabel(v: symbol, label: symbol)
+// Definitely does not have this label.
+// This is dual of `hasLabel`. We need a better name.
```

Comment:
If this is necessary (I'm guessing as a strong override to the weaker `mayHaveLabel`?), I'd like to propose `cannotHaveLabel` instead.

Reply:
`cannotHaveLabel` seems like a good name instead of `mayNotHaveLabel`.

BTW, please read this rules with a grain of salt. I am not positive that the rules for `mayNotHaveLabel` and `hasLabel` are completely correct. Datalog has [stratification restrictions](https://stackoverflow.com/questions/12379775/datalog-stratification) when using negation. I was introducing some of these relations willy nilly to get souffle to do something.